server/data/welcomeClient.ts

```diff
@@ -55,6 +57,8 @@ export default class WelcomeClient {
         toDate: toDate.format('YYYY-MM-DD'),
         query: searchQuery,
       },
+      timeout: 15000,
+      retryCount: 0,
     }) as Promise<PaginatedResponse<RecentArrival>>
```

Comment:
default timeout is 5 seconds and 3 retries. 

When BASM "struggles" the highest duration we've seen is 8 seconds. 
This would trigger a retry so the user is unlikely to see anything for at least 15 seconds. 

For this situation, the new config values would mean that the user sees the list in 8 seconds. 
Disabling retries as a user is reasonably likely to have manually retried after 15 seconds.